### DISCUSSION TITLE
Table 1

### DISCUSSION THREAD

**Discussion by morgan-hannah** [IBAHCM/RPiR#529] [Registration]
These are the repositories for table 1 - Morgan, Lotta, Beth and Fernanda

Morgan Hannah's: https://github.com/SBOHVM/morganhannahSeries01.git

**Comment by 2394135R** [comment]
Here's mine :)

https://github.com/SBOHVM/lottaruhaSeries1.git

-Lotta

**Reply by morgan-hannah** [reply]
Great code!

**Comment by FernandaSanchezR** [comment]
My repository - Fernanda Sanchez
https://github.com/SBOHVM/fernandasanchezrSeries01.git

**Reply by 2394135R** [reply]
Hi @FernandaSanchezR, your repository looks great :)

**Reply by morgan-hannah** [reply]
Looks great!

**Comment by bethccore** [comment]
Our table has @morgan-hannah @2394135R @FernandaSanchezR and me (@bethccore)

**Comment by bethccore** [comment]
My repository: https://github.com/SBOHVM/bethccoreSeries01.git

- Beth

**Reply by 2394135R** [reply]
Hi @bethccore, amazing repository 🥇

**Reply by morgan-hannah** [reply]
Runs perfectly!

**Comment by 2394135R** [comment]
Hi @morgan-hannah, I have taken a look at your repository, and your code looks beautiful!

**Comment by FernandaSanchezR** [comment]
Hello! My repository for series 2. @morgan-hannah @bethccore @2394135R 
https://github.com/SBOHVM/FernandaSanchezRSeries02.git

**Comment by 2394135R** [comment]
Hi guys! Here is the link for my Series 2 repository. @morgan-hannah @FernandaSanchezR @bethccore  https://github.com/SBOHVM/lottaruhaSeries02.git

**Comment by bethccore** [comment]
Hey guys! I know I've already raised my repository to you guys via mentioning you in an issue comment (thanks for the responses thus far!:)), but I'm going to put it in here too just in case it's needed. So here is my Series 2 repository (again) @2394135R @FernandaSanchezR @morgan-hannah 

https://github.com/SBOHVM/bethccoreSeries02.git

**Comment by morgan-hannah** [comment]
Hi guys here is my repository for series 2 https://github.com/SBOHVM/morganhannahSeries02.git @2394135R @FernandaSanchezR @bethccore 
It is missing 2.4, but the rest should be there

**Reply by 2394135R** [reply]
Hi @morgan-hannah, I tried to click the link, and it did not work. Is your repository public yet?

**Reply by morgan-hannah** [reply]
Oh oops! Should be fixed now!

**Reply by 2394135R** [reply]
Thank you, it works perfectly now :)
